Example/android/build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
     ext {
         buildToolsVersion = "27.0.3"
         minSdkVersion = 16
-        compileSdkVersion = 27
+        compileSdkVersion = 28
```

Comment:
I don't really know why it is required. In other PRs everything is passing without this change 🤔

Reply:
1. It's because the version react-native of Example is changes, from `0.58.0-rc.2` to `"0.58.0-rc.2` https://github.com/kmagiera/react-native-gesture-handler/pull/406/files#diff-77190fabae122eec07dd4211d79a25f7R20
2. And because of that, the Example taking latest react-native version, which is `0.58.3` the latest release (6 days ago from now).
3. In 0.58.0, there's already an update from android support 0.27 to 0.28 https://github.com/facebook/react-native/commit/fe6f6cd46f077b96733f67b6e3d6769cdfb5ee21#diff-4a7ba4beeeab016621279eed94a6270bL315
4. Because the android support 0.28 is being used, we need to change the compileSdkVersion to 28.

Reply:
ok, thanks!